DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -95,7 +95,7 @@ public function process(ContainerBuilder $container)
 
                 $groupDefaults[$resolvedGroupName]['items'][] = array(
                     'admin'        => $id,
-                    'label'        => '',
+                    'label'        => !empty($attributes['label']) ? $attributes['label'] : '',
```

Comment:
assuming the service definition looks like this:
```
<service id="sonata.admin.post" class="Acme\DemoBundle\Admin\PostAdmin">
    <tag name="sonata.admin" manager_type="orm" group="Content" label="Post"/>
        <argument />
        <argument>Acme\DemoBundle\Entity\Post</argument>
        <argument />
        <call method="setTranslationDomain">
        <argument>AcmeDemoBundle</argument>
    </call>
</service>
```

https://github.com/wodka/SonataAdminBundle/blob/issue-3018-admins-sort/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L83

on line 89 it will set the "group" label to "Content" and without my change it will ignore the "entry" label on line 98 - I think it should then be set to "Post"